### PR TITLE
Add target for creating disk-images

### PR DESCRIPTION
**Pull request by javipolo**

### CODE REVIEW

File: training/README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ In order to run accelerated AI workloads, we've prepared [bootc](https://github.
 | intel           | Create bootable container for Intel Habanalabs platform             |
 | nvidia          | Create bootable container for NVidia platform                       |
 | vllm            | Containerized inference/serving engine for LLMs                     |
+| disk-amd        | Create disk image from bootable container for AMD platform          |
+| disk-intel      | Create disk image from bootable container for Intel platform        |
+| disk-nvidia     | Create disk image from bootable container for Nvidia platform       |
```
Comment (rhatdan) [inline training/README.md]: Please alphabetize.

File: training/README.md [hunk 2]
```diff
@@ -91,6 +94,38 @@ To add cloud-init to your existing bootc container image, executing make cloud-<
...
+| Variable              | Description                       | Default                                          |
+|-----------------------|-----------------------------------|--------------------------------------------------|
+| DISK_TYPE             | Type of image to build            | `qcow2`                                          |
+| IMAGE_BUILDER_CONFIG  | Path to a build-config file       | `EMPTY`                                          |
+| BOOTC_IMAGE           | Image to use as input             | `$REGISTRY/$REGISTRY_ORG/$IMAGE_NAME:$IMAGE_TAG` |
+
+Image builder config file is documented in [bootc-image-builder README](https://github.com/osbuild/bootc-image-builder?tab=readme-ov-file#-build-config)
+
+The following image disk types are currently available:
+| Image type            | Target environment                                                                    |
```
Comment (rhatdan) [inline training/README.md]: Should probably be DISK type.

File: training/common/Makefile.common [hunk 3]
```diff
@@ -68,6 +75,28 @@ growfs:
...
 	mkdir -p build; cp -R ../common/usr build
 
+.PHONY: bootc-image-builder
+bootc-image-builder:
+	mkdir -p build/store
+	podman run \
+	  --rm \
+	  -ti \
+	  -v $(GRAPH_ROOT):/var/lib/containers/storage \
+	  $(ARCH:%=--arch %) \
```
Comment (rhatdan) [inline training/common/Makefile.common]: Does this work with the ARCH removed above?

I don't think you ever want to do this?  This should only be handled in the bootc section, you should not run a non native bootc-image-builder.

File: training/common/Makefile.common [hunk 4]
```diff
@@ -68,6 +75,28 @@ growfs:
...
+	  $(AUTH_JSON:%=-v %:/run/containers/0/auth.json) \
+	  $(IMAGE_BUILDER_CONFIG:%=-v %:/config.json) \
+	  --privileged \
+	  --pull newer \
+	  -v ./build:/output \
+	  -v ./build/store:/store \
+	  $(BOOTC_IMAGE_BUILDER) \
+	  $(ARCH:%=--target-arch %) \
```
Comment (rhatdan) [inline training/common/Makefile.common:92]: This one is GOOD. Noth the one above.

File: training/README.md (hunk 1 above)
Comment (javipolo) [reply training/README.md]: Oops, done :)

File: training/common/Makefile.common (hunk 3 above)
Comment (javipolo) [reply training/common/Makefile.common]: Ok, removing then ... I copied most of it from the Makefile.common target that is present in the /recipes section
https://github.com/containers/ai-lab-recipes/blob/d3ba2b62fbf46432fb017e2d777bec53df318fd2/recipes/common/Makefile.common#L108-L130

Comment (rhatdan) [reply training/common/Makefile.common]: Need to fix this there also.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**